controllers/apply/lib/field-types/day-month.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+const moment = require('moment');
+const Joi = require('../joi-extensions');
+
+const Field = require('./field');
+
+class DayMonthField extends Field {
+    getType() {
+        return 'day-month';
+    }
+
+    defaultSchema() {
+        return Joi.dayMonth().required();
+    }
+
+    defaultMessages() {
+        return [
+            {
+                type: 'base',
+                message: this.localise({
+                    en: 'Enter a day and month',
+                    cy: 'Rhowch ddiwrnod a mis',
+                }),
+            },
+            {
+                type: 'any.invalid',
+                message: this.localise({
+                    en: 'Enter a real day and month',
+                    cy: 'Rhowch ddiwrnod a mis go iawn',
+                }),
+            },
+        ];
+    }
+
+    get displayValue() {
+        if (this.value) {
+            const dt = moment({
```

Comment:
This copies the previous `formatDayMonth` method in `formatters.js`. Eventually that file should be removed.